$NetBSD: patch-bf,v 1.1 2007/11/01 13:08:06 rillig Exp $

Adding "const" to a primitive type is useless. Sunpro doesn't like it.

--- src/CurrentWindowCmd.cc.orig	2007-03-19 20:57:09.000000000 +0100
+++ src/CurrentWindowCmd.cc	2007-11-01 13:05:58.247018000 +0100
@@ -111,7 +111,7 @@ FluxboxWindow &WindowHelperCmd::fbwindow
     return *FocusControl::focusedFbWindow();
 }
 
-MoveCmd::MoveCmd(const int step_size_x, const int step_size_y) :
+MoveCmd::MoveCmd(int step_size_x, int step_size_y) :
   m_step_size_x(step_size_x), m_step_size_y(step_size_y) { }
 
 void MoveCmd::real_execute() {
@@ -120,7 +120,7 @@ void MoveCmd::real_execute() {
       fbwindow().y() + m_step_size_y);
 }
 
-ResizeCmd::ResizeCmd(const int step_size_x, const int step_size_y) :
+ResizeCmd::ResizeCmd(int step_size_x, int step_size_y) :
     m_step_size_x(step_size_x), m_step_size_y(step_size_y) { }
 
 void ResizeCmd::real_execute() {
@@ -134,7 +134,7 @@ void ResizeCmd::real_execute() {
     fbwindow().resize(w, h);
 }
 
-MoveToCmd::MoveToCmd(const int step_size_x, const int step_size_y, const unsigned int refc) :
+MoveToCmd::MoveToCmd(int step_size_x, int step_size_y, unsigned int refc) :
     m_step_size_x(step_size_x), m_step_size_y(step_size_y), m_refc(refc) { }
 
 void MoveToCmd::real_execute() {
@@ -161,7 +161,7 @@ void MoveToCmd::real_execute() {
 }
 
 
-ResizeToCmd::ResizeToCmd(const int step_size_x, const int step_size_y) :
+ResizeToCmd::ResizeToCmd(int step_size_x, int step_size_y) :
     m_step_size_x(step_size_x), m_step_size_y(step_size_y) { }
 
 void ResizeToCmd::real_execute() {
